.storybook/test-runner.ts

```diff
@@ -0,0 +1,26 @@
+import { waitForPageReady, TestRunnerConfig } from '@storybook/test-runner';
+
+import { toMatchImageSnapshot } from 'jest-image-snapshot';
+
+const config: TestRunnerConfig = {
+  setup() {
+    expect.extend({ toMatchImageSnapshot });
+  },
+  async postVisit(page, context) {
+    const elementHandler = await page.$('#storybook-root');
+    const innerHTML = await elementHandler?.innerHTML();
+    if (!innerHTML) throw new Error('No innerHTML found');
+    expect(innerHTML).toMatchSnapshot();
```

Comment:
Some resources generate an automatic ID, which changes on each render, so matching the HTML to a snapshot might be unfeasible.

Reply:
Yeah, ive been tinkering with some regex to replace those with dynamic id's. But it started to become slightly insane. 

For now, we can ignore these tests (6ish) and can add sufficient alternative testing for these and any others that are ID generating